frontend/freebase2wikidata.js

```diff
@@ -981,7 +981,30 @@ $(function() {
             }
           } else {
             // New object
-            createNewStatement(property, freebaseObject);
+            var isDuplicate = false;
+            for (var c = 0; c < wikidataClaims[property].length; c++) {
+              var wikidataObject = wikidataClaims[property][c];
+              var wikidataObjQid = wikidataObject.mainsnak.datavalue.value.id;
```

Comment:
What if the datavalue type of the Wikidata claim is not EntityId but e.g. string?